### DISCUSSION TITLE
[求助]求一位擅长数学的绳匠

### DISCUSSION THREAD

**Discussion by xibozhi** [share121/inter-knot#981] [其他]
![屏幕截图 2024-08-10 200242](https://github.com/user-attachments/assets/ffdde70f-15fb-45c8-a232-a3ec04d3fa65)
本人准初三，数学一般，之前做题的时候发现了这个
（如图，有直角三角形ABC，将其绕点B旋转α°得到与之全等的直角三角形DBE，连接CE，在旋转过程中发现直线AD始终平分线段CE)
用geogebra画了一遍貌似是成立的，但对着图想了半天没想出来原因。
如果我的猜测没问题的话希望绳匠们给一个证明，如果我是错的也希望各位指出，

**Comment by qwq1949** [comment]
```cpp
作直线 EG 平行于 AC，交射线 AF 于 M
连接 EM
```
###### 有平分线就先向延长平分那条线段的线，然后证全等
###### 但这里是证明它的 所以可以反着来 （因为反证初中不给分qwq）
```cpp
∵ AC // EM 
易证 △ACF ∽ △MEF
```
###### 这个初三会讲
###### 这样就很容易想到只需要证全等就可以了
###### 要证全等 只需要证明任意一对边 这里考虑证明 AC = EM
###### 因为 AC = DE，所以证明 DE = EM 就可以证明全等了
###### 证明 DE = EM 需要证明 ∠EDM = ∠EMD
###### 又因为 ∠EMD = ∠CAF，所以证明 ∠CAF = ∠EDM 
```cpp
DAB + ABC + ACB + CAF = 180 = ADB + BDE + EDM
ACB + ABC = 90 = BDE ， BAD = BDA
∴ ∠CAF = ∠EDM 
然后顺着证就行了
```

**Reply by xibozhi** [reply]
谢谢楼上的大佬，待会试试

**Reply by qwq1949** [reply]
好了qwq

**Comment by wwwwsn147** [comment]
你可以先去了解下手拉手模型

**Reply by xibozhi** [reply]
ok

**Comment by Elysia-Ego-093** [comment]
![IMG_20240810_211403](https://github.com/user-attachments/assets/0294398c-e853-4378-94e3-f8253aff608e)
如图，作EG//AC交AF于点G，然后证三角形ACF和EGF全等就行了

**Comment by dont-continue** [comment]
这题可以利用倍长中线，使E点向下到射线AF做条线段EH使EH平行于AC，然后就可以试着去证明三角形ACF全等于三角形HEF然后就是CF=FE然后结果就出来了（同学平时可以去了解一下拓展的数学知识对于平时写作业有很大的帮助同为准初三我建议你空余时间去看看相似三角形这个对于平时写作业有很大的帮助可以给你节省很多数形结合上的麻烦(σ′▽‵)′▽‵)σ）

**Reply by dont-continue** [reply]
嗯，我说的可能有点潦草了但大体就是倍长中线，然后去证明他全等基本上是这样

**Reply by xibozhi** [reply]
ok谢谢

**Comment by Blazing-dust** [comment]
嗯呢嗯呢~（重点在于利用全等的知识证相等，得中点，上面的人讲得很详细，我就来水一下）

**Comment by lllliufeng** [comment]
坏了我高三毕业差不多不会了
![image](https://github.com/user-attachments/assets/671466e6-4ba2-4332-a80b-86fca27b9517)
因为 AB=AD,AC=AE,∠BAD=∠CAE
所以 Rt△ABC=Rt△ADE
所以 ∠ABD=∠ACE
由 与△ABF和△ACF有公共边且公共边对应的对角相等
可知 存在一个圆使得BAFC四点在圆上且AC为圆的直径
由于 F在圆上
所以 AF⊥CF（直径对直角）
又因为 AC=AE
所以 F为CE中点（三线合一）
或许这样也行？

**Reply by silverwolf510trillion** [reply]
请教大佬，”由与△ABF和△ACF有公共边且公共边对应的对角相等可知，存在一个圆使得BAFC四点在圆上且AC为圆的直径“这一步是什么原理？

**Reply by lllliufeng** [reply]
> 请教大佬，”由与△ABF和△ACF有公共边且公共边对应的对角相等可知，存在一个圆使得BAFC四点在圆上且AC为圆的直径“这一步是什么原理？

四点共圆的判定之一
若线段**同侧二点到线段两端点连线夹角相等**，那么这两点和线段两端点这四点共圆

证明如下：
△ABF在某个圆上，
若C在圆内，∠ACF>∠ABF，不合题
若C在圆外，∠ACF<∠ABF，不合题
若C在圆上，∠ACF=∠ABF，合题
所以C在这个圆上，BAFC四点共圆

**Comment by WASDDestroy** [comment]
**【证】：** 直线AD平分线段CE，理由如下：

由题意，ΔABC≌ΔDBE

∴∠ABC=∠DBE, AB=BD, BC=BE

∴∠ABD=∠CBE

∴ΔABD∽ΔCBE

∴∠ADB=∠CEB

∵∠ADB+∠BDF=180°

∴∠CEB+∠BDF=180°, 即四边形BDFE为圆内接四边形

∵∠BDE=90°

∴BE为上述圆的一条直径

取BE中点O，由上述证明知O即为圆心。

连结OF

∵O为圆心

∴OF=OE

∴∠OFE=∠OEF

∵BC=BE

∴∠BCE=∠BEC

∴∠OFE=∠BCE

同位角相等得到OF∥BC。

∵O为BE中点

∴OF为ΔBCE的一条中位线

由中位线的性质，F即为CE中点，AF（直线AD）平分线段CE。

证明完毕。

* * *

（准高三，已经忘光了准初三已经学了哪些内容了，所以就按照自己第一时间想到的方法直接写了一版证明）

感谢楼主让我想起了美好的初中数学（以及我一直传承到现在的又臭又长过程）

**Comment by shilijia-siweikongjian** [comment]
还是不难想到用倍长中线的逆运用的。
![Screenshot_20240810_220401](https://github.com/user-attachments/assets/0b7d5028-dec2-4815-9f04-7c4dbe895ebc)
思路大概是证得角5=角6，等角对等边，即可得到全等。在作用手拉手相似，四点共圆之类导一导角，就可以证出来～～刚中考完，死去的记忆突然复苏～～
